Sparks_Steven/Assignments/Flask Olympics/olympics6/templates/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+  <p>{{gold}}</p>
+    <form class="" action="/process" method="POST">
+		<input type="hidden" name="building" value="farm" />
+	    <input type="submit" name="farm" value="Find Gold!"/>
+	    <input type="hidden" name="building" value="cave" />
+	    <input type="submit" name="cave" value="Find Gold!"/>
+	    <input type="hidden" name="building" value="house" />
+	    <input type="submit" name="house" value="Find Gold!"/>
+	    <input type="hidden" name="building" value="casino" />
+	    <input type="submit" name="casino" value="Find Gold!"/>	
+    </form>
```

Comment:
This probably won't work. You've declared the same variable "building" multiple times with different values.

If this were Python, it would be identical to doing the following:

x = 'cave'
x = 'house'
x = 'casino'

etc.

At the end of the day, even if you define x to = a hundred different things, the only one that will remain is the very last one. To get this working properly, you have to separate each of these into separate forms. Take a look at some of the ninjagold assignment submissions (the views) in my repo to see how to break it out.